scripts/incident_schema_v3.json

```diff
@@ -12,6 +12,7 @@
                 "Picture"
             ],
             "plural_title": "Photos",
+            "propertyOrder": 1,
```

Comment:
Since `propertyOrder` will be used to determine order for showing these sections, here's the desired order in which people should be entering information:
 * Incident Details [0]
 * Vehicles [1]
 * People [2]
 * Photos [3]